testsuite/pytests/test_connect_one_to_one.py

```diff
@@ -21,11 +21,10 @@
 
 import numpy as np
 import unittest
-import test_connect_helpers as hf
-from test_connect_parameters import TestParams
+import connect_test_base as hf
```

Comment:
I find the name `hf` totally not understandable. Why not just
```suggestion
from connect_test_base import ConnectTestBase
```
...

Reply:
You are right, will fix it.